oc2ls.md

```diff
@@ -369,22 +369,22 @@ OpenC2 data types are defined using an abstract notation that is independent of
 | :--- | :--- |
 | **Primitive Types** |   |
 | Binary | A sequence of octets.  Length is the number of octets. |
-| Boolean | A logical entity that can have two values: `true` and `false`. |
+| Boolean | An element with one of two values: `true` and `false`. |
 | Integer | A whole number. |
 | Number | A real number. |
-| Null | Nothing, used to designate fields with no value. |
-| String | A sequence of characters. Each character must have a valid Unicode codepoint.  Length is the number of characters. |
+| Null | An unspecified or non-existent value. |
```

Comment:
This description is very different than previous. Why is this being changed?

Reply:
1. For consistency with the forthcoming JADN spec https://github.com/davaya/openc2-jadn-1/blob/working/jadn-v1.0-wd01.md.

2. JADN says that because Null when serialized in JSON messages is the literal `null`, representing a non-existent value, whereas Null when used as a Type means that the type is unspecified.

If you are sending a message about the Autobahn with a speed limit field, the message would be:
```
{
  "speed_limit": null
}
```
The JSON value is the non-existent literal `null` and the speed limit represented by that message is unspecified.